stroyprombeton/tests/tests_views.py

```diff
@@ -947,6 +947,46 @@ def test_filter_options_by_tags(self):
         self.assertFalse(returned.intersection(disappeared))
 
 
+@tag('fast', 'catalog')
+class SeriesMatrix(BaseCatalogTestCase):
+    fixtures = ['dump.json']
+
+    def get_page_url(self):
+        page = CustomPage.objects.get(slug='series')
+        return page.url
+
+    def get_page(self, *args, **kwargs):
+        return self.client.get(self.get_page_url(*args, **kwargs))  # Ignore CPDBear
```

Comment:
You should edit signature for `get_page_url` 
```python
    def get_page_url(self, *args, **kwargs):
```

Reply:
removed redundant `*args, **kwargs` passes instead